install/kubernetes/helm/meshery-operator/charts/meshery-cilium/README.md

```diff
@@ -0,0 +1,46 @@
+# meshery-cilium
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
```

Comment:
How do we get rid of this static versioning? Does the helm chart releaser overwrite this?

Reply:
For my past experiences, helm/charts repo has the static checking. Every PR need to nump the Helm charts version. The CI error if not. So, i'm not sure there exist any tool can help on automatic syncing the charts or application version. 


And these `README.md` we use the tool to created automaticly. It's means we can only focus on update helm charts.

Reply:
Gotcha. ✔️